flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/sink/FileSink.java

```diff
@@ -580,6 +584,11 @@ public T withRollingPolicy(CheckpointRollingPolicy<IN, String> rollingPolicy) {
             return self();
         }
 
+        public T disableLocalWriting() {
+            this.writerConfig.put("fs.hdfs.no-local-write", "true");
```

Comment:
It might be nicer to change this string literal into a static final constant of FileSink, and use the same constant in HadoopFileSystem.

Reply:
Thanks for your suggestion, I have modified this field.